tests/apps/pubsub-subscriber/app.go

```diff
@@ -138,12 +154,14 @@ func subscribeHandler(w http.ResponseWriter, r *http.Request) {
 		w.WriteHeader(http.StatusInternalServerError)
 		json.NewEncoder(w).Encode(appResponse{
 			Message: errorMessage,
+			Status:  "DROP",
 		})
 		return
 	}
 
 	json.NewEncoder(w).Encode(appResponse{
-		Message: "success",
+		Message: "consumed",
+		Status:  "SUCCESS",
```

Comment:
This is needed because pubsub expects a Status field if the response if available from subscriber. https://github.com/dapr/dapr/blob/master/pkg/runtime/runtime.go#L1105